R/Orders.R

```diff
@@ -78,37 +78,44 @@ orders <-
     if (.is_id) {
       # if it's an order_id
       .o_id <- symbol_id
+      .url <- get_url(
+        pacth = c("orders", .o_id),
+        query = list(nested = nested),
+        live = live
+      )
+    } else if (isTRUE(client_order_id)) {
+      # if it's a client order id
+      .url <- get_url(
+        path = "orders:by_client_order_id", 
+        query = list(client_order_id = symbol_id), 
+        live = live
+      )
     } else {
-      # if its a ticker symbol
-      .o_id <- NULL
+      # if it's a ticker symbol(s)
       symbol_id <- toupper(symbol_id)
+      if(length(symbol_id) > 1) paste(symbol_id, collapse = ',')

```

Comment:
Not sure what this is doing since it's not assigned

Reply:
My bad, this is meant to collapse a vector of tickers into a single comma-separated string, per the API docs: 
_symbols
string
A comma-separated list of symbols to filter by (ex. “AAPL,TSLA,MSFT”). A currency pair is required for crypto orders (ex. “BTCUSD,BCHUSD,LTCUSD,ETCUSD”)._

Will assign this and pass it to the query.

Reply:
Perfect!